capture-sdk/sdk/src/main/java/net/gini/android/capture/AmountCurrency.kt

```diff
@@ -1,5 +1,18 @@
 package net.gini.android.capture
 
+/**
+ * Enum class represents currently supported currencies
+ *
+ * <p>
+ * Currency codes expressed in ISO 4217 standard </p>
+ *
+ * <ul>
+ *  <li> EUR - stands for EURO (official currency of the euro-zone)
+ *  <li> GBP - stands for the British pound sterling
+ *  <li> USD - stands for United States Dollar
+ *  <li> CHF - stands for Confoederatio Helvetica franc
+ * </ul>
```

Comment:
[Request] Please add dots to the end of the sentences.

Also you can use markdown with kdoc so we can get rid of html tags.